django_perf_rec/api.py

```diff
@@ -112,7 +112,10 @@ def save_or_assert(self):
         orig_record = self.records_file.get(self.record_name, None)
 
         if orig_record is not None:
-            assert self.record == orig_record, "Performance record did not match for {}".format(self.record_name)
+            assert self.record == orig_record, "Performance record did not match for {}\n{}".format(
```

Comment:
True, you can be technically correct by using a var (don't put it on sys): http://doc.pytest.org/en/latest/example/simple.html#detect-if-running-from-within-a-pytest-run

You also can't use a conftest.py file, you'll have to create `django_perf_rec.pytest_plugin` and put an entry point in setup.py like https://github.com/adamchainz/pytest-randomly/blob/master/setup.py#L64